README.md

```diff
@@ -74,10 +74,11 @@ services:
     image: soundbot
     volumes:                                           
     - ./sounds:/app/sounds                             
-    - ./config.json:/app/config/config.json            
+    - ./config/config.json:/app/config/config.json
     - ./db.json:/app/db.json                           
 ```
-+ Create your `config.json` in the same directory
++ Create a `config/config.json` file as described above.
++ Create an empty `db.json` file.
```

Comment:
Turns out there is no way to make the Docker volume mount create this file for you, since it will always create directories.  An option is to move `db.json` into a `db` folder, but that would break compatibility.